pkgs/by-name/mo/morph/package.nix

```diff
@@ -30,6 +31,9 @@ buildGoModule rec {
     mkdir -p $lib
     cp -v ./data/*.nix $lib
     wrapProgram $out/bin/morph --prefix PATH : ${lib.makeBinPath [ openssh ]};
+    installShellCompletion --cmd morph \
+      --bash <($out/bin/morph --completion-script-bash) \
+      --zsh <($out/bin/morph --completion-script-zsh)
```

Comment:
This needs to be conditional, see #308283

Reply:
You can also use an emulator when available, like in [vrc-get](https://github.com/NixOS/nixpkgs/blob/34e731d3f4620f6bee24a43c11f2baf815f1f5f0/pkgs/tools/misc/vrc-get/default.nix#L27-L38)